run.sh

```diff
@@ -35,10 +35,12 @@ prepare () {
   # These directiories essentially define the structure of our API endpoints.
   # TODO: These should perhaps live in python, near the schemas (defined in api/)?
   STATES_DIR="${API_OUTPUT_DIR}/";
+  INPUT_BASE_DIR="${API_OUTPUT_DIR}/";
   # TODO: I think deploy_dod_dataset.py may currently have an implicit
   # requirement that the county model JSON is in a /county subdirectory of the
   # states?
   COUNTIES_DIR="${API_OUTPUT_DIR}/county";
+  STATES_OUTPUT_DIR="${API_OUTPUT_DIR}/state";
```

Comment:
1. Should this be `${API_OUTPUT_DIR}/us/states/` to match https://www.dropbox.com/scl/fi/o4bec2kz8dkcxdtabtqda/CAN-V1-API-Draft.paper?dl=0&eid=&oref=e&rlkey=f7elx3zmo6tt5s7mj5nvap9a0&t=278234600567106294963247#:t=278234600567106294963247 ?
2. STATES_DIR vs STATES_OUTPUT_DIR is confusing.  Maybe rename this to `V1_STATES_DIR`?
3. `COUNTIES_DIR` is the legacy directory. Perhaps we should create `V1_COUNTIES_DIR` that points at `us/counties` and move the top-100 counties list to there?

Reply:
I guess overall i found it a bit confusing that the input and output directories were linked? i was trying to figure out a way to separate them

Reply:
FWIW- This is why I wanted to generate the new schema JSON files directly from the pandas dataframes or whatever instead of going through this intermediate directory (which is both the output directory for the legacy schema and the input directory for this new stuff that you're adding).  I don't have a great suggestion for fixing the naming other than hopefully we get rid of the intermediate directory eventually, so we maybe don't need to worry about it too much.

Reply:
i changed it around a lilttle bit in my pr lmk what you think